stumpy/aamp_motifs.py

```diff
@@ -129,6 +129,10 @@ def _aamp_motifs(
             motif_distances.append(query_matches[:max_matches, 0])
             motif_indices.append(query_matches[:max_matches, 1])
 
+        if len(query_matches) == 0:  # pragma: no cover
+            # create fake `query_matches`
```

Comment:
Maybe "create empty `query_matches`" is slightly better?

Reply:
@seanlaw 
Just to make sure you did not mistype anything... so I should use the term "empty"? (because the array being empty was the main issue). But, if that is okay with you, I can go ahead and change it for sure. (I had the term `pseudo` in mind but I think it is not suitable here)

Reply:
Or maybe just omit the comment altogether?

Reply:
Sure.